gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/writer/PolicyWriter.java

```diff
@@ -93,13 +97,23 @@ private PolicyMetadata createPolicyMetadata(final Bundle bundle, final Bundle ra
             policyMetadata.setSubtag(policyEntity.getSubtag());
         }
         Set<Dependency> dependencies = getPolicyDependencies(folderableEntity.getId(), rawBundle);
+        Set<Dependency> filteredDependencies = dependencies.stream().filter(dependency -> {
```

Comment:
Now it is looking good. 
Can you please move this logic into a separate method called **getFilteredPolicyDependencies**. So that it becomes like below:
Set<Dependency> dependencies = getFilteredPolicyDependencies(getPolicyDependencies(...